### PR TITLE
[Test only] - Add test to secure Static - Current User functionality.

### PR DESCRIPTION
**Pull request by KarinG**
Overview
----------------------------------------
Test only!

Before
----------------------------------------
No specific test for Static - Current User

After
----------------------------------------
We have a test ✅

### CODE REVIEW

File: tests/src/FunctionalJavascript/ContactSubmissionTest.php [hunk 1]
```diff
@@ -224,6 +224,65 @@ public function testStaticAndAutocompleteOnContactElement() {
...
+  public function testStaticCurrentUser() {
+    $this->drupalLogin($this->rootUser);
+    $this->drupalGet(Url::fromRoute('entity.webform.civicrm', [
+      'webform' => $this->webform->id(),
+    ]));
+    // The label has a <div> in it which can cause weird failures here.
+    $this->assertSession()->waitForText('Enable CiviCRM Processing');
+    $this->assertSession()->waitForField('nid');
+    $this->getSession()->getPage()->checkField('nid');
+    $this->assertSession()->assertWaitOnAjaxRequest();
```
Comment (jitendrapurohit) [inline tests/src/FunctionalJavascript/ContactSubmissionTest.php]: We have a helper for the above 4 lines - `$this->enableCivicrmOnWebform();`. Can we use that pls? :)

File: tests/src/FunctionalJavascript/ContactSubmissionTest.php [hunk 2]
```diff
@@ -224,6 +224,65 @@ public function testStaticAndAutocompleteOnContactElement() {
...
+
+    // Scenario: root user is configuring the form - so that the logged in Contact details appear
+    $this->assertSession()->checkboxChecked('Existing Contact');
+    $this->assertSession()->checkboxChecked('First Name');
+    $this->assertSession()->checkboxChecked('Last Name');
+
+    $this->getSession()->getPage()->pressButton('Save Settings');
+    $this->assertSession()->pageTextContains('Saved CiviCRM settings');
+    $this->assertPageNoErrorMessages();
```
Comment (jitendrapurohit) [inline tests/src/FunctionalJavascript/ContactSubmissionTest.php]: Similarly, can we use `$this->saveCiviCRMSettings();` for the above 3 lines?

**Comment by KarinG** [comment]
Absolutely - ok I have make the updates.

**Comment by jitendrapurohit** [comment]
Thanks @KarinG

**Pull request merged**